### PR TITLE
Support for indirect patch events

### PR DESCRIPTION
**Pull request by jkodumal**
Supports large feature patches by accepting indirect events, which are pointers to the feature flag to be updated

### CODE REVIEW

File: requestor.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
+package ldclient
```
Comment (jkodumal) [inline requestor.go]: We needed a re-usable way to make feature flag requests from the eval API.

**Comment by jkodumal** [comment]
One thing I'm still working out is how this is going to work with the TTL. If you make a change, and the stream sends a message suggesting a refresh, it won't make sense to receive the TTL-respecting value.

**Comment by jkodumal** [comment]
See here: https://github.com/launchdarkly/foundation/pull/33

Will create a new PR incorporating the new foundation abstraction.

**Comment by jkodumal** [comment]
Hadn't meant to close this-- the client-side changes are independent of the foundation changes I made.

**Comment by pkaeding** [comment]
lgtm

**Comment by pkaeding** [comment]
:+1:

**Pull request merged**